build.gradle

```diff
@@ -2,15 +2,18 @@
 buildscript {
 
     ext.kotlin_version = "1.5.31"
+
     repositories {
 
         google()
         mavenCentral()
     }
+
     dependencies {
 
         classpath 'com.android.tools.build:gradle:7.1.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.android.support:multidex:1.0.3'
```

Comment:
Why we need multidex, do we have over 64K methods? or is there any spec req?

Reply:
Exactly!